packages/scss/src/components/viewTabs/component.scss

```diff
@@ -0,0 +1,69 @@
+@mixin component($atRoot: 'without: rule') {
+	padding: 0;
+	margin: 0;
+	list-style-type: none;
+	display: flex;
+	flex-wrap: wrap;
+	width: var(--components-viewTabs-width);
+	flex-direction: var(--components-viewTabs-direction);
+	box-shadow: 0 0 0 var(--commons-divider-width) var(--palettes-grey-300);
+	border-radius: var(--components-viewTabs-borderRadius);
+	background-color: var(--colors-white-color);
+	font-size: var(--components-viewTabs-fontSize);
+	line-height: var(--components-viewTabs-lineHeight);
+
+	@at-root ($atRoot) {
+		.viewTabs-item {
+			flex-grow: 1;
+			position: relative;
+
+			&::before,
+			&::after {
+				content: '';
+				border-width: 0;
+				inset: var(--components-viewTabs-dividerInset);
+				border-color: var(--palettes-grey-300);
+				position: absolute;
+				border-style: solid;
+			}
+
+			&::before {
+				right: calc(var(--commons-divider-width) * -1);
+				border-left-width: var(--commons-divider-width);
+				left: auto;
+			}
+
+			&::after {
+				bottom: calc(var(--commons-divider-width) * -1);
+				border-top-width: var(--commons-divider-width);
+				top: auto;
+			}
+		}
+
+		.viewTabs-item-tab {
+			width: 100%;
+			border: 0;
+			margin: 0;
+			padding: var(--components-viewTabs-padding);
+			background-color: var(--components-viewTabs-background);
+			box-shadow: 0 0 0 var(--commons-divider-width) var(--components-viewTabs-shadow);
+			border-radius: var(--components-viewTabs-borderRadius);
+			gap: var(--spacings-XS);
+			color: var(--components-viewTabs-color);
+			text-align: center;
+			justify-content: center;
+			display: flex;
+			align-items: center;
+
+			position: relative;
+			z-index: 1;
+			cursor: var(--components-viewTabs-cursor);
+			font: inherit;
+		}
+
+		.viewTabsPanel {
```

Comment:
Why do we need to have a specific element as container?

Reply:
Panels are embedded in the component to manage their deactivated and focused states.